test/commands/profiles/create.test.js

```diff
@@ -60,30 +60,12 @@ describe('commands', () => {
         });
 
       createTest()
-        .nock('https://api.twilio.com', api => {
-          api.get(`/2010-04-01/Accounts/${constants.FAKE_ACCOUNT_SID}.json`).reply(200, {
-            sid: constants.FAKE_ACCOUNT_SID
-          });
-          api.post(`/2010-04-01/Accounts/${constants.FAKE_ACCOUNT_SID}/Keys.json`).reply(200, {
-            sid: constants.FAKE_API_KEY,
-            secret: constants.FAKE_API_SECRET
-          });
-        })
         .do(ctx => {
           sinon.stub(os, 'hostname').returns('X'.repeat(64));
-          return ctx.testCmd.run();
+          ctx.returned = ctx.testCmd.getApiKeyFriendlyName();
         })
         .it('truncates apiKeyFriendlyName to 64 characters', ctx => {
-          expect(ctx.stdout).to.equal('');
-          expect(ctx.stderr).to.contain(helpMessages.AUTH_TOKEN_NOT_SAVED);
-          expect(ctx.stderr).to.contain('Saved default.');
-          expect(ctx.stderr).to.contain('configuration saved');
-          expect(ctx.stderr).to.contain(
-            `Created API Key ${constants.FAKE_API_KEY} and stored the secret using libsecret`
-          );
-          expect(ctx.stderr).to.contain(
-            `See: https://www.twilio.com/console/runtime/api-keys/${constants.FAKE_API_KEY}`
-          );
+          expect(ctx.returned.length).to.equal(64);
```

Comment:
I'd prefer to test the exact string match since I was confused at first why `X` was repeating 64 times and matches the length of the returned friendly name, but really the numbers are not directly related.

Reply:
Fair enough